content/community/_index.md

```diff
@@ -3,3 +3,53 @@ title: "Community"
 headless: true
 opened: true
 ---
+
+The IPFS community is as decentralized and distributed as IPFS itself — our community members are globally located, active online and in person, and using and building on IPFS in a virtually endless number of ways. You can get more information about meetings, events, apps people are building, and more using the resources listed here.
+
+## Discussion/Support Forum
+
+Join the conversation on the [https://discuss.ipfs.io/](IPFS Forums) for everything from technical questions, project brainstorming, talk on IPFS use cases and more.
+
+## Research Discussion
+
+The [IPFS Collaborative Notebook for Research](https://github.com/ipfs/notes) repo tracks discussions on topics related to IPFS research, ideas and proposals for new systems/features, and other items that haven't yet been tracked into a specific project.
+
+## Events Calendar
+
+Find all of the open community video calls for the IPFS project in the [IPFS Community events calendar](https://calendar.google.com/calendar/embed?src=ipfs.io_eal36ugu5e75s207gfjcu0ae84@group.calendar.google.com).
+
+## Community Tools
+
+Join the discussion and documentation on IPFS community practices in the [community tools repo](https://github.com/ipfs/community/). If you're interested in starting, helping with or expanding IPFS awareness in your local geographical or online community, you can find tools and info here.
```

Comment:
Not sure what "join the discussion and documentation" means here. Also lmk if my suggestion for the last sentence messed with its meaning!

```suggestion
Join the discussion and documentation on IPFS community practices in the [community tools repo](https://github.com/ipfs/community/). If you want to start or get involved with a community chapter to help expand IPFS awareness in your local geographical or online community, here's where to look.
```